Client/FluentClient.cs

```diff
@@ -96,7 +96,7 @@ public virtual IRequest SendAsync(HttpRequestMessage message)
 
             // clone the underlying message because HttpClient doesn't normally allow re-sending
             // the same request, which would break IRequestCoordinator.
-            return new Request(message, this.Formatters, request => this.BaseClient.SendAsync(request.Message.Clone(), request.CancellationToken), this.Filters.ToArray())
+            return new Request(message, this.Formatters, request => this.BaseClient.SendAsync(request.Message.Clone(), request.CancellationToken), this.Filters)
```

Comment:
This makes the request filters a reference to the client filters, so changes to the request filters will unexpectedly be global. For example:

```cs
// one client filter
Console.WriteLine(client.Filters.Length);

// remove request filter
IRequest request = await client.GetAsync(…);
request.Filters.Remove<DefaultErrorFilter>();
IResponse response = await request;

// zero client filters
Console.WriteLine(client.Filters.Length);